voronoi-diagram/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for voronoi-diagram 1.0.1
+// Project: https://github.com/mikolalysenko/voronoi-diagram
+// Definitions by: Michael Neu <https://github.com/michaelneu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "voronoi-diagram" {
```

Comment:
Use an external module style: Move `function voronoi(points: Point[]) : VoronoiDiagram;` and `export = voronoi;` out of this wrapper (to the top level) and change the wrapper to `declare namespace voronoi`. This gets merged with `function voronoi` so that you can access the types as in `import voronoi = require("voronoi"); const x: voronoi.Point = [1, 2]`.